.env.example

```
@@ -71,6 +71,7 @@ DOCKER_SCRIPT_EXECUTOR="/bin/bash -c" # Path to the script executor to use (defa
 DOCKER_CPU_LIMIT=2 # Number of CPUs to limit the container to (see Docker documentation for more info)
 DOCKER_MEMORY_LIMIT=2g # Memory limit for the container (see Docker documentation for more info)
 DOCKER_CONTAINER_AUTOREMOVE=false # Disable auto-removal to retain container logs post-execution (default: false)
+DOCKER_IMAGE_PULL_POLICY=always # always | if-not-present | never. "always" contacts the registry once per step even when the image is already local, which costs ~2.2s per step for a small image and more for a large one; "if-not-present" skips that but never picks up a moved tag (default: always)
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DOCKER_IMAGE_PULL_POLICY key should go before the DOCKER_MEMORY_LIMIT key

(UnorderedKey)